src/bean/ui/scratch.cljs

```diff
@@ -0,0 +1,46 @@
+(ns bean.ui.scratch
+  (:require [bean.scratch :as scratch]
+            [bean.ui.util :refer [cs]]))
+
+(defn set-eval-state [sheet]
+  (assoc-in sheet
+            [:ui :scratch-evaluation-state]
+            (if (:errors sheet)
+              :error
+              :evaluated)))
+
+(defn set-eval-state-pending [sheet]
+  (assoc-in sheet [:ui :scratch-evaluation-state] :pending))
+
+(defn text-area [sheet]
+  [:div {:class (cs (str "scratch-state-"
+                         (name (or (get-in @sheet [:ui :scratch-evaluation-state])
+                                   :evaluated)))
```

Comment:
Should put `scratch-evaluation-state ` `evaluated` by default instead of defaulting here.